README.md

```diff
@@ -24,9 +24,9 @@ To build Spark and its example programs, run:
 
     build/mvn -DskipTests clean package
 
-(You do not need to do this if you downloaded a pre-built package.)
+(You do not need to do this if you have downloaded a pre-built package.)
```

Comment:
I don't agree with most of these changes. This was grammatical. The rest is just alternate wording which I don't think is meaningfully better.